clients/src/main/java/org/apache/kafka/clients/Metadata.java

```diff
@@ -127,6 +127,15 @@ public synchronized void addTopics(String... topics) {
     }
 
     /**
+     * Add one or more topics to maintain metadata for
+     */
+    public synchronized void addTopics(Iterable<String> topics) {
```

Comment:
No, not really. I think at first I made it iterable in the hope that I'd be able to delegate addTopics(String...) to this method, but that didn't work. I left it since it didn't seem to make much difference.